community/en/BlackScholes_tf_pricing.ipynb

```diff
@@ -1,25 +1,39 @@
 {
```

Comment:
<p>Please change the urls to match the appropriate link.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/examples/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I've updated the urls to point at the link where the notebook _will be_ once merged to master. Is it right way to do?

Reply:
Yes. Thanks.